Makefile

```diff
@@ -399,6 +399,18 @@ fv fv/latency.log: $(REMOTE_DEPS) image-test bin/iptables-locker bin/test-worklo
 fv-bpf:
 	$(MAKE) fv FELIX_FV_ENABLE_BPF=true
 
+KO_DIR := "/lib/modules/$(shell uname -r)/"
+WIREGUARD_KO_PATH := $(shell find $(KO_DIR) -name "wireguard.ko")
+fv-wireguard:
```

Comment:
When running in CI, I recommend having a force-enable flag that disables the auto-detection, then make sure it's run on a platform that supports it.  We don't want the FV to silently fail in CI, even if it's failing because the autodetection failed.

Reply:
@realgaurav - if this issue is resolved, please mark it so. I'd like for @robbrockbank to help prioritize getting this PR merged.